lib/exception_notifier.rb

```diff
@@ -47,20 +54,22 @@ def breadcrumbs(data:, message: nil)
       raise ArgumentError.new('data must be a hash') unless data.is_a?(Hash)
       raise ArgumentError.new('when providing a message, it must be a string') if message.present? && !message.is_a?(String)
 
-      error_client.breadcrumbs.record do |crumb|
-        crumb.message = message
-        crumb.data = data
-      end
+      crumb = breadcrumb.new(message: message, data: data)
+      error_client.add_breadcrumb(crumb)
     end
 
     def last_breadcrumb
-      error_client.breadcrumbs.buffer.last
+      error_client.get_current_scope&.breadcrumbs&.peek
```

Comment:
This is not in their official documentation. see: https://github.com/getsentry/sentry-ruby/blob/b1771ff3edeb23cb6a3cce1464cb6577cf90f75c/sentry-ruby/lib/sentry/breadcrumb/sentry_logger.rb#L58